HIP/hip-1000.md

```diff
@@ -0,0 +1,161 @@
+---
+hip:1000
+title: HIP Unlimited token associations per Account
+author: Anirudh Ghanta <anirudh.ghanta@hedera.com>
+working-group: a list of the technical and business stakeholders' name(s) and/or username(s), or name(s) and email(s).
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft <Draft | Review | Last Call | Active | Inactive | Deferred | Rejected | Withdrawn | Accepted | Final | Replaced>
+created: 2022-02-17
+discussions-to: <a URL pointing to the official discussion thread>
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
```

Comment:
I think all 4 of these are removed? Unless there was a previous HIP that discussed token associations, in which case we should add a link here to say this one replaces it.